src/topology/continuous_function/compact.lean

```diff
@@ -188,6 +188,28 @@ instance : normed_ring C(α,R) :=
 { norm_mul := λ f g, norm_mul_le (mk_of_compact f) (mk_of_compact g),
   ..(infer_instance : normed_group C(α,R)) }
 
+lemma continuous_const [nonempty α] : continuous (continuous_map.const : R → C(α, R)) :=
+begin
+  rw metric.continuous_iff,
+  intros a ε hε,
+  refine ⟨ε/2, (show 0<ε/2, by linarith), λ b hb, _⟩,
+  rw dist_eq_norm at hb ⊢,
+  refine lt_of_le_of_lt _ (show ε/2 < ε, by linarith),
+  rw continuous_map.norm_eq_supr_norm,
+  apply csupr_le,
+  intro x,
+  apply le_of_lt,
+  simp [hb],
+end
+
+instance [nonempty α] : has_continuous_smul R C(α, R) :=
```

Comment:
This should be done at least for a normed algebra.

Reply:
Maybe it's even true for `has_continuous_mul R`?

Reply:
I don't think continuous functions are specified for `[empty α]`(this is more of a notation, I think). I have done it for normed algebras and `[nonempty α]`.

Reply:
Can you please try moving this to `continuous_function.algebra` (and not using the normed structure)?

Reply:
This lemma was no longer needed since it is the same as `continuous_const'` .